packages/gestalt/src/TapArea.js

```diff
@@ -119,9 +119,18 @@ const TapAreaWithForwardRef: React$AbstractComponent<unionProps, unionRefs> = fo
     },
   );
 
+  const handleOnKeyPress = (event) => {
+    // Check to see if space or enter were pressed
+    if (!disabled && onTap && keyPressShouldTriggerTap(event)) {
+      // Prevent the default action to stop scrolling when space is pressed
+      event.preventDefault();
+      onTap({ event, disableOnNavigation: () => {} });
```

Comment:
Is this casting the type of disableOnNavigation? If so, I think we want `void` instead of `{}`

Reply:
no, this is setting `disableOnNavigation` to  `() => {}`, not a type, but the value

Reply:
Oh I see, sometimes hard to read between object literals and renamed destructuring and Flow declarations and… 😅